src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -33,18 +33,18 @@
  * </p>
  */
 public class DriveTrain extends SubsystemBase {
-  private SpeedControllerGroup m_leftSide, m_rightSide;
-  private CANSparkMax m_leftFront, m_leftBack, m_rightFront, m_rightBack;
-  private Encoder m_leftEncoder, m_rightEncoder;
+  private final SpeedControllerGroup m_leftSide, m_rightSide;
+  private final CANSparkMax m_leftFront, m_leftBack, m_rightFront, m_rightBack;
+  private final Encoder m_leftEncoder, m_rightEncoder;
 
-  private Solenoid m_highGear;
+  private final Solenoid m_highGear;
+ 
+  private final AHRS m_gyro;
```

Comment:
Looks like since this the gyro is initialized in a try/catch, it doesn't like being final.

Can you try initializing it with a temporary? Something like this (Line 78):
```
    AHRS gyro;
    try {
      gyro = new AHRS(SPI.Port.kMXP);
    } catch (Exception e) {
      gyro = null;
      e.printStackTrace();
    }
    m_gyro = gyro;
```